config/schedule.rb

```diff
@@ -1,3 +1,5 @@
+env 'PATH', '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
```

Comment:
I guess, although nearly every other aspect of how the app is deployed on a particular server is also in this repository.

It turns out that "configur[ing] cron itself" is a bit complicated. The path has to be set in the crontab that whenever is using, which is, of course, under the control of whenever (and capistrano). This way, we can be sure that the path and the tasks are always both in the same place.

Neither is great, but the alternative is that someone has to log into production, type `sudo crontab -e` and then add exactly the same line that whenever adds with this fix.

Technically speaking, 😬 I think only the "first" application on the server needs this fix... which is an argument in your favor, I guess?